library/public/camera.h

```diff
@@ -58,6 +58,7 @@ class F3D_EXPORT camera
    */
   virtual camera& dolly(double val) = 0;
   virtual camera& pan(double right, double up, double forward = 0) = 0;
+  virtual camera& zoom(double factor) = 0;
```

Comment:
some doc about the diff between zoom and dolly is important here imo.

Reply:
do we copy-paste the descriptions from corresponding `vtkCamera` methods?

Reply:
that seems a bit overkill. I'm just suggesting sthing to help libf3d user figure out which method they should use between dolly and zoom if they are not super familiar with the concepts. Also "factor" is a bit opaque.

Reply:
> Also "factor" is a bit opaque.

Damn, should have stuck with `val` which is much better :D

The reasoning was that we basically have `dolly(x){ distance /= x }` and `zoom(x){ angle /= x }` where the argument is a multiplier (or rather the inverse of) meaning that the values should be around `1` (with `1` being a fixed-point/no-op). As opposed to `roll(x)`, `yaw(x)`, ... where the parameter is more like a delta that gets added.

Anyways, I'll add individual doc for these methods. What's the proper doxygen format to use here?